vagrant/roles/subman-devel/tasks/main.yml

```diff
@@ -176,6 +176,43 @@
     line: "PYTHONPATH={{ subman_checkout_dir }}/src:{{ subman_checkout_dir }}/python-rhsm/src"
   become: yes
 
+- name: compile c artifacts
```

Comment:
Please name this task `compile c artifacts (debian only)` so that playbook output will be more obvious why this gets skipped for other distros.